code/go/0chain.net/blobbercore/filestore/chunk_writer_test.go

```diff
@@ -0,0 +1,92 @@
+package filestore
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestWrite(t *testing.T) {
+
+	fileName := filepath.Join(os.TempDir(), "testwrite_"+strconv.FormatInt(time.Now().Unix(), 10))
+
+	content := "this is full content"
+
+	w, err := NewChunkWriter(fileName)
+	if err != nil {
+		assert.Fail(t, "failed to create ChunkWriter", err)
```

Comment:
Use require rather than assert, as we want the test to fail on an error.  Same below.